grafana/docker-compose.yml

```diff
@@ -2,31 +2,31 @@ version: "3.7"
 services:
   grafana:
     image: grafana/grafana:latest
-    container_name: grafana
     ports:
-      - 8080:3000
+      - "8080:3000"
```

Comment:
Just use a range notation on this, docker will auto assign the range

- "8080-8082:3000"